rosa_release_notes/rosa-release-notes.adoc

```diff
@@ -1,87 +1,41 @@
 :_content-type: REFERENCE
 [id="rosa-release-notes"]
-= {product-title} 4.10 release notes
+= What's new with {product-title}
 include::_attributes/attributes-openshift-dedicated.adoc[]
 :context: rosa-release-notes
 
 toc::[]
 
 {product-title} (ROSA) is a fully-managed, turnkey application platform that allows you to focus on delivering value to your customers by building and deploying applications. Red Hat and AWS site reliability engineering (SRE) experts manage the underlying platform so you do not have to worry about the complexity of infrastructure management. ROSA provides seamless integration with a wide range of AWS compute, database, analytics, machine learning, networking, mobile, and other services to further accelerate the building and delivering of differentiating experiences to your customers.
 
-[id="about-this-release-section_{context}"]
-== About this release
+{product-title} clusters are available at https://console.redhat.com/openshift. With the Red Hat OpenShift Cluster Manager application for ROSA, you can deploy {product-title} clusters to either on-premises or cloud environments.
 
-{product-title} https://access.redhat.com/errata/RHEA-2022:2252[RHEA-2022:2252] is now available. This release uses Kubernetes 1.24 with CRI-O runtime. New features, changes, and known issues that pertain to {product-title} 4.10 are included in this topic.
+[id="updating_{context}"]
+== Updating the ROSA CLI tool
 
-{product-title} clusters are available at https://console.redhat.com/openshift. With the {cluster-manager-first} application for ROSA, you can deploy {product-title} clusters to either on-premise or cloud environments.
+You can download the latest version of the ROSA CLI from the Hybrid Cloud Console.
 
-[id="about-this-release_{context}"]
-== ROSA layered and dependent component support and compatibility
-The scope of support for layered and dependent components of ROSA changes independently of the ROSA version. To determine the current support status and compatibility for an add-on, see the xref:../rosa_architecture/rosa_policy_service_definition/rosa-life-cycle.adoc#rosa-life-cycle[{product-title} update life cycle].
+To update to the latest version of the ROSA CLI:
 
-[id="rosa-new-features_{context}"]
-== New features and enhancements
+. Download the file from the https://console.redhat.com/openshift/downloads[Hybrid Cloud Console].
+. Unzip the downloaded file.
+. In the terminal, type `sudo mv rosa /usr/bin/rosa` then press **Enter**.
+//. In the terminal, type `chmod a+x /usr/bin/rosa` to make the ROSA binary you downloaded executable.
+. Type `rosa version` in the terminal to confirm your version. A note appears saying "Your ROSA CLI is up to date." when you've updated.
```

Comment:
Per ISG, Do not use _appear_ unless there is no alternative.
Maybe:
When the update is complete, the "Your ROSA CLI is up to date." is displayed.

Watch the _you've_ if you rewrite.